CreeDictionary/res/layouts/verb-ai-full.layout

```diff
@@ -44,7 +44,7 @@ tooltips:
 | "I"                   | Ind+Prt+1Sg                         | PV/e+*+Cnj+Prt+1Sg        |
 | "you (one)"           | Ind+Prt+2Sg                         | PV/e+*+Cnj+Prt+2Sg        |
 | "s/he"                | Ind+Prt+3Sg                         | PV/e+*+Cnj+Prt+3Sg        |
-| "we (but not you)     | Ind+Prt+1Pl                         | PV/e+*+Cnj+Prt+1Pl        |
+| "we (but not you)"    | Ind+Prt+1Pl                         | PV/e+*+Cnj+Prt+1Pl        |
```

Comment:
it broke shit

Reply:
So what broke here and why? (I hope this comment gets associated in the right place)

Reply:
Oops, sorry, see a missing quote. Understand now.